README.md

```diff
@@ -116,3 +116,15 @@ You may be able to get away without thinking about all this if your project spec
 Error suggestions may be triggered from some underlined compiler warnings. There
 is no visual indication, currently this will basically be for 'import' warnings,
 and can be triggered by 'alt-enter' (PureScript: Show Quick Fixes).
+
+## Hacking on atom-ide-purescript
+
+Install dependencies
+  • `bower install`
+  • `npm install`
+
+Build: `pulp build`
```

Comment:
No, remove `pulp build` only. `npm install` will actually trigger `npm run bundle`, which is all you need to build each time (it includes `pulp build` as part of a command line), but usually I'd run that directly.